From: Andrew Morton <akpm@linux-foundation.org>
Subject: memcg-mem_cgroup_get_from_ino-returns-null-on-error-fix
Date: Mon Jan 12 03:36:40 PM PST 2026

build fix

mm/memcontrol.c: In function 'mem_cgroup_get_from_id':
mm/memcontrol.c:3624:35: error: 'ino' undeclared (first use in this function); did you mean 'int'?
 3624 |         cgrp = cgroup_get_from_id(ino);
      |                                   ^~~
      |                                   int

Cc: SeongJae Park <sj@kernel.org>
Cc: Shakeel Butt <shakeel.butt@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/memcontrol.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/mm/memcontrol.c~memcg-rename-mem_cgroup_ino-to-mem_cgroup_id-fix-fix
+++ a/mm/memcontrol.c
@@ -3621,7 +3621,7 @@ struct mem_cgroup *mem_cgroup_get_from_i
 	struct cgroup_subsys_state *css;
 	struct mem_cgroup *memcg = NULL;
 
-	cgrp = cgroup_get_from_id(ino);
+	cgrp = cgroup_get_from_id(id);
 	if (IS_ERR(cgrp))
 		return NULL;
 
_
